drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -180,6 +180,11 @@ matterManufacturer:
     vendorId: 0x1339
     productId: 0x006B
     deviceProfileName: light-color-level-2000K-7000K
+  - - id: "4921/125"
```

Comment:
It looks like there is an extra '-' on this line

Reply:
Thank you!!! I have resolved this @nickolas-deboom !

Reply:
Thank you! I approved the PR